packages/camera/tizen/src/permission_manager.cc

```diff
@@ -4,10 +4,13 @@
 
 #include "permission_manager.h"
 
+#ifndef TV_PROFILE
 #include <privacy_privilege_manager.h>
+#endif
 
 #include "log.h"
 
+#ifndef TV_PROFILE
```

Comment:
`log.h` is only used by `PermissionToString` so maybe these `#endif` and `#ifndef` are redundant?